doc/devel/dependencies.rst

```diff
@@ -414,6 +414,9 @@ Required
   * `dvipng <https://ctan.org/pkg/dvipng>`_
   * `underscore <https://ctan.org/pkg/underscore>`_
 
+* if using a minimal install of TeX-Live, you may also need ``collection-fontsrecommended``
```

Comment:
I would put this under the TeX-Live bullet.  "A LaTeX distribution that includes at least a, b, c, e.g., ..."